_posts/0000-01-02-Lvyuanba.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "yuanba,Welcome to our second slide!"
+---
+Your text
```

Comment:
> this is new message

**### i got u**
###  **_shopping list_**
- [ ] wireless mouse

- [ ] the mask